pkg/controller/clone-controller.go

```diff
@@ -161,7 +163,13 @@ func (cc *CloneController) processPvcItem(pvc *v1.PersistentVolumeClaim) error {
 		}
 
 		cc.raisePodCreate(pvcKey)
-		sourcePod, err = CreateCloneSourcePod(cc.clientset, cc.image, cc.pullPolicy, clientName, pvc)
+
+		podResourceRequirements, err := GetDefaultPodResourceRequirements(cc.cdiClient)
```

Comment:
So this is more a stylistic thing than a code problem. podResourceRequirements is looked up here, just so we can pass it to CreateCloneSourcePod. I think it makes sense to look up the podResourceRequirements in CreateCloneSourcePod, but that will require us to pass cc.cdiClient to CreateCloneSourcePod.

Reply:
Thanks for your comments :) We updated commit applying this style